toolchain/check/cpp/import.cpp

```diff
@@ -919,8 +920,8 @@ static auto ImportClassObjectRepr(Context& context, SemIR::ClassId class_id,
                              context, class_type_inst_id, field_type_inst_id),
                          .name_id = field_name_id,
                          .index = SemIR::ElementIndex(fields.size())}));
-    context.clang_decls().Add(
-        {.decl = decl->getCanonicalDecl(), .inst_id = field_decl_id});
+    auto key = SemIR::ClangDeclKey::NonFunctionDecl(decl);
```

Comment:
Does it work to use the `field` value here, or does that give a bad result for `IndirectFieldDecl`? i.e.:
```suggestion
    auto key = SemIR::ClangDeclKey(field);
```

Reply:
That doesn't work for indirect fields, added a comment to help future readers.